setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name=pytest-marker-bugzilla
 summary=py.test bugzilla integration plugin, using markers
-description-file=README.txt
```

Comment:
Is there any other use for this README.txt file ?
If there is not, I would remove it